types/react-dom/index.d.ts

```diff
@@ -68,3 +68,7 @@ export function unstable_renderSubtreeIntoContainer<P>(
     element: ReactElement<P>,
     container: Element,
     callback?: (component?: Component<P, ComponentState> | Element) => any): Component<P, ComponentState> | Element | void;
+export function unstable_createPortal(
+    children: ReactNode,
+    container: Element,
+    key?: string): ReactElement<any>;
```

Comment:
ReactNode is a type alias, so I'm not sure whether it's possible to extend it. (stackoverflow suggested it isn't)

I could, however create a `ReactPortalElement` interface that extends `ReactElement<{}>` The former could reside in the react-dom package.